go-controller/pkg/ovn/policy.go

```diff
@@ -222,8 +243,100 @@ func (oc *Controller) deleteACLAllow(namespace, policy, logicalSwitch,
 	}
 }
 
-func (oc *Controller) addACLDeny(namespace, logicalSwitch,
-	logicalPort, policyType string) {
+func (oc *Controller) addIPBlockACLDeny(namespace, policy, logicalSwitch,
+	logicalPort, policyType, except, priority string) {
+	var match, l3Match, direction, portMatch string
```

Comment:
Would you mind changing portMatch to lportMatch at these places too?

Reply:
Done